docs/.project/INTEGRATION_TEST.md

```diff
@@ -92,11 +92,6 @@ Based on the [test diagram](./test.bpmn.png):
 * [ ] add a regular expression (`^CAM-[0-9]+$`) to the invoice number field
 * [ ] save file on disk as `test.form`
 * [ ] file imports correctly after save
-* [ ] Set the __Execution Platform Version__ to `Camunda 8 (Zeebe 1.0)`
-  * [ ] 4 errors are shown: `Text`, `Number`, `Checkbox`, and `Radio` are not supported
-  * [ ] Clicking on an error focuses the respective element
-* [ ] Set the __Execution Platform Version__ to `Camunda 8 (Zeebe 1.2)`
-  * [ ] 0 errors are shown
```

Comment:
I'd rather expect the integration test to still validate that the feature works, but with different execution platform versions.

Reply:
E.g. an `Expression` field is rather new and not supported in 8.4